modules/management/kubectl-apply/main.tf

```diff
@@ -23,6 +23,11 @@ locals {
   apply_manifests_map = tomap({
     for index, manifest in var.apply_manifests : index => manifest
   })
+
+  install_kueue         = try(var.kueue.install, false)
+  install_jobset        = try(var.jobset.install, false)
```

Comment:
IIUC, jobset is a dependency for Kueue to work. Why not automatically install jobset if queue is requested. 

```
  install_jobset        = try(var.jobset.install, local.install_kueue)
```

There is precedence for this sort of thing in startup-script module where we automatically install ansible if the user wants to run an ansible script.

Note: I still think there is value to have a separate variable just for jobset (if you only want to install jobset, or control the version)